core/root/config/templates/cluster.yaml

```diff
@@ -1443,6 +1437,12 @@ experimental:
   # This is intended to be used in combination with .controller.iam.role.name. See #297 for more information.
   kiamSupport:
     enabled: false
+    image: 
+      repo: quay.io/uswitch/kiam
+      tag: v2.8
+      rktPullDocker: false
+    server: localhost:443
+    agent: kiam-server:443
```

Comment:
Maybe worth clarifying these as something like this?
```yaml
serverAddresses:
  server: localhost:443
  agent: kiam-server:443
```

Reply:
Looks pretty reasonable. Will address this later today